src/foolscap/test/test_gifts.py

```diff
@@ -349,8 +351,9 @@ def _checkBob(res):
     # introduction pattern is going to change soon, so it has been disabled
     # until I figure out what the new scheme ought to be asserting.
 
-    def OFF_bobGotCarol(self, (balice,bcarol)):
-        if self.debug: print "Bob got Carol"
+    def OFF_bobGotCarol(self, args):
+        balice,bcarol = args        
+        if self.debug: print("Bob got Carol")
```

Comment:
minor nit, but would be nice to be consistent to use `a, b` i.e. pep8 style with a space after the comma (as in other places)

Reply:
Fixed.